frontend/pages/dashboard.vue

```diff
@@ -1,25 +1,260 @@
+          <div id="search" class="relative mr-2.5" v-tooltip.top="t('dashboard.tooltip.search')">
+            <input type="text" ref="searchInput" @input="searchJourneys" v-model="searchValue"
+              class="rounded-3xl bg-input dark:bg-input-dark placeholder-input-placeholder dark:placeholder-text-light-dark border px-3 py-1.5 border-border-grey dark:border-input-dark focus:outline-none focus:ring-1 focus:ring-cta-border "
+              :placeholder="t('dashboard.search')">
+            <button @click="searchInput.focus()">
+              <SvgSearchIcon class="absolute top-1 right-1 w-7 h-7" />
+            </button>
+          </div>
+          <div id="filter" class="mr-4">
+            <SvgFilterIcon @click="toggle" aria-haspopup="true" aria-controls="overlay_tmenu"
+              class="w-9 h-9 hover:cursor-pointer" />
+            <div class="">
+              <TieredMenu ref="menu" id="overlay_tmenu" :model="items" popup class="rounded-xl"
+                :pt="{ menuitem: { class: 'hover:bg-cta-bg rounded-md' }, content: { class: 'hover:bg-cta-bg rounded-md' } }">
+                <template #start>
+                  <h1 class="text-sm ml-2 text-input-placeholder">
+                    <T keyName="dashboard.sort.header" />
+                  </h1>
+                  <Divider type="solid" class="text-[#CCCCCC] border-b mt-1 mb-1" />
+                </template>
+                <template #item="{ item, props, hasSubmenu }">
+                  <a v-ripple class="flex align-items-center hover:bg-cta-bg-light rounded-md text-text text-sm"
+                    v-bind="props.action">
+                    <span :class="item.icon"></span>
+                    <span class="ml-2">{{ item.label }}</span>
+                    <i v-if="hasSubmenu" class="pi pi-angle-right ml-auto"></i>
+                  </a>
+                </template>
+              </TieredMenu>
+            </div>
+          </div>
+        </div>
+        <NuxtLink to="/journey/new" class="mr-2.5 hidden lg:flex flex-row items-center">
+          <button
+            class="bg-cta-bg dark:bg-cta-bg-fill border-2 border-cta-border dark:border-cta-bg-fill text-text py-1 px-4 rounded-xl font-semibold flex flex-row">
+            <SvgCreateNewJourneyIcon class="w-5 h-5 mr-2 fill-text" />
+            <T keyName="dashboard.new" />
+          </button>
+        </NuxtLink>
+        <NuxtLink to="/settings">
+          <SvgSettingsIcon class="w-9 h-9 -mt-1" />
+        </NuxtLink>
+      </div>
+    </div>
+    <div id="header-mobile-second-row" class="flex lg:hidden mt-3 justify-between">
+      <div id="search-and-filter" class="flex flex-row">
+        <div id="filter" class="mr-2">
+          <SvgFilterIcon @click="toggle" aria-haspopup="true" aria-controls="overlay_tmenu"
+            class="w-9 h-9 hover:cursor-pointer" />
+          <div class="">
+            <TieredMenu ref="menu" id="overlay_tmenu" :model="items" popup class="rounded-xl"
```

Comment:
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/807a7d18-7514-4601-bcd2-b15219a64ab3)
yes i know es ist mein fehler im sorry ich merk es nur hiernochmal an damit du es nicht vergisst in figma ist die neue vorlage. bitte die auch für das bearbeiten der reise benutzten